docs/sources/administration/view-server/_index.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "View server"
+weight = 100
++++
+
+# View server information
+
+This setting contains information about tools that Grafana Server Admins can use to learn more about their Grafana servers.
```

Comment:
Is this a setting or a capability?

Reply:
This is collecting three extant topics that all fall under the general category of "see what is going on with your Grafana instance."